src/pages/Profile.js

```diff
@@ -238,7 +238,7 @@ const FenceLink = ({ provider, displayName }) => {
    * Render
    */
   const isBusy = isLoadingStatus || isLoadingAuthUrl || isLinking
-  const expireTime = addDays(30, parseISO(issuedAt)) // can't use parseJSON because dates don't specify time offset
+  const expireTime = addDays(30, parseJSON(issuedAt))
```

Comment:
Curious  - did the version update cause a breakage?

Reply:
No, `parseJSON` is preferred because it's lighter; the version update gave it the ability to parse dates without offsets.